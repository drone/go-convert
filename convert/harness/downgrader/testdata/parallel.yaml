kind: pipeline
spec:
  stages:
  - name: test
    spec:
      steps:
      - spec:
          steps:
          - name: test1
            spec:
              image: busybox:latest
              run: |-
                echo "Do a test here"
                echo "For example run a test suite"
            type: script
          - name: test2
            spec:
              image: busybox:latest
              run: |-
                echo "Do another parallel test here"
                echo "For example run a lint test"
            type: script
        type: parallel
    type: ci
version: 1
