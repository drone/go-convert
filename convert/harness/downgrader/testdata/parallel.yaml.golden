pipeline:
  identifier: default
  name: default
  orgIdentifier: default
  projectIdentifier: default
  properties:
    ci:
      codebase:
        build: <+input>
  stages:
  - stage:
      identifier: test
      name: test
      spec:
        cloneCodebase: true
        execution:
          steps:
          - parallel:
            - step:
                identifier: test1
                name: test1
                spec:
                  command: |-
                    echo "Do a test here"
                    echo "For example run a test suite"
                  image: busybox:latest
                timeout: ""
                type: Run
            - step:
                identifier: test2
                name: test2
                spec:
                  command: |-
                    echo "Do another parallel test here"
                    echo "For example run a lint test"
                  image: busybox:latest
                timeout: ""
                type: Run
        platform:
          arch: Amd64
          os: Linux
        runtime:
          spec: {}
          type: Cloud
      type: CI
